client/my-sites/upgrades/domain-search/domain-search.jsx

```diff
@@ -149,13 +144,19 @@ var DomainSearch = React.createClass( {
 			<Main className={ classes }>
 				<SidebarNavigation />
 				{ content }
+				<QueryProductsList />
 			</Main>
 		);
 	}
-} );
+}
 
-module.exports = connect(
+export default connect(
 	( state ) => ( {
-		domainsWithPlansOnly: currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY )
+		selectedSite: getSelectedSite( state ),
+		selectedSiteId: getSelectedSiteId( state ),
+		selectedSiteSlug: getSelectedSiteSlug( state ),
+		domainsWithPlansOnly: currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY ),
+		isSiteUpgradeable: siteId => isSiteUpgradeable( state, siteId ),
```

Comment:
One other thing: can we cache the `isSiteUpgradeable`, so they would be created only once for each `siteId`, instead of being created every time?